cmd/experimental/kjobctl/pkg/cmd/testing/fake.go

```diff
@@ -34,10 +37,11 @@ import (
 type TestClientGetter struct {
 	util.ClientGetter
 
-	k8sClientset     k8s.Interface
-	kueueClientset   kueueversioned.Interface
-	kjobctlClientset kjobctlversioned.Interface
-	dynamicClient    dynamic.Interface
+	k8sClientset       k8s.Interface
+	kueueClientset     kueueversioned.Interface
+	kjobctlClientset   kjobctlversioned.Interface
+	dynamicClient      dynamic.Interface
+	UnstructuredClient resource.RESTClient
```

Comment:
```suggestion
	unstructuredClient resource.RESTClient
```

Can we use `With...` instead?

Reply:
Done